crates/l2/monitor/mod.rs

```diff
@@ -1,8 +1,7 @@
 // TODO: Handle this expects
 #![expect(clippy::expect_used)]
-#![expect(clippy::panic)]
 #![expect(clippy::indexing_slicing)]
-
+#[allow(clippy::result_large_err)]
```

Comment:
Nit. Use `expect` instead of `allow` for clippy lints. This way, if the lint no longer applies, clippy will let you know that it is safe to remove. 
```suggestion
#[expect(clippy::result_large_err)]
```